playbooks/openstack-cloud-controller-manager-acceptance-test-lb-octavia-nested/run.yaml

```diff
@@ -0,0 +1,85 @@
+- hosts: all
+  become: yes
+  pre_tasks:
+    - shell:
+        cmd: |
+          set -e
+          set -x
+          cat << EOF >> /tmp/dg-local.conf
+          [[local|localrc]]
+          NETWORK_GATEWAY=10.0.0.1
+          disable_service q-agt q-svc q-dhcp q-l3 q-meta q-metering c-api c-bak c-sch c-vol ceilometer-acentral ceilometer-acompute ceilometer-alarm-evaluator ceilometer-alarm-notifier ceilometer-anotification ceilometer-api ceilometer-collector cinder s-account s-container s-object s-proxy
```

Comment:
LbaaS server have been defined in roles, we can instead of these line by following logic:
```yaml
    - role: create-devstack-local-conf
      enable_services:
        - 'lbaas'
```

Reply:
we but we should disable swift, since the 8080 port is used both k8s and swift.

Reply:
Alternative we can add some logic in create-devstack-local-conf for k8s specified enable/disable services list, I think we should put all of these service setting logic into same place, more readable and easy to maintenance.

And swift port also can be changed in dg-local.conf
https://github.com/openstack-dev/devstack/blob/master/lib/swift#L56-L57

Reply:
I personally think it is more readable of my current way, it is flexible to config for specify jobs and the run path is: pre_tasks -- > roles -- > tasks -- > post_tasks. don't need to check the create-devstack-local-conf which will be more complex if more jobs in, and there will be a long condition checking chain the role definition.
but OK for me to change to use that role.